src/main/java/com/volmit/iris/util/decree/annotations/Param.java

```diff
@@ -28,15 +28,15 @@
 public @interface Param {
     String DEFAULT_DESCRIPTION = "No Description Provided";
 
+    String REQUIRED = "Required";
+
     /**
      * The main name of this command.<br>
      * Required parameter.<br>
      * This is what is used in game, alongside any (if specified) {@link #aliases() aliases}
      */
     String name();
 
-    boolean required() default false;
```

Comment:
I changed this for a reason, why change it back? We need default AND required. Yes i know they collide with each other but it doesnt cause any issues, where as this solution, removing required and ASSUMING required if the default isnt there, does not cover all edge cases.

Reply:
See above